server-adapters/resteasy-reactor-netty/src/main/java/org/jboss/resteasy/plugins/server/reactor/netty/ChunkOutputStream.java

```diff
@@ -48,9 +48,7 @@
 
     private final ReactorNettyHttpResponse parentResponse;
 
-    static final Throwable RESPONSE_WRITE_ABORTED_ON_CANCEL = new WriterException("Response write aborted due to cancellation");
-
-    static final Throwable RESPONSE_WRITE_ABORTED_ON_DISCARD = new WriterException("Response write aborted due to discarded bytes");
+    static final Throwable RESPONSE_WRITE_ABORTED = new WriterException("Response write aborted abruptly");
```

Comment:
I still don't think this should be static. To me the stack trace would look very odd as it's not coming from where the exception is actually thrown.